DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: connectapi
 Title: Utilities for Interacting with the 'Posit Connect' Server API
-Version: 0.4.0
+Version: 0.4.0.9000
```

Comment:
Do we have on our checklist(s) for release to bump this after the release? I thought it was on the `usethis` one, but if not we should do that so we don't have to tack these onto PRs like this

Reply:
Agreed. It isn't on our checklist, and I don't *think* it's on the `usethis` one either.

Reply:
There are a few outdated things on the `usethis` issue template that don't fit so I was thinking of just making an issue template for release prep or something.

Reply:
Could also send a PR to usethis too if it's general?

Reply:
Yeah. I had been considering that for other items -- like there are some list items that refer to packages that are (as far as I can tell) no longer available.